Assets/Scripts/UIAndMenus/EscMenuController.cs

```diff
@@ -1,10 +1,15 @@
+using GameStateManagement;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine;
+using UnityEngine.InputSystem;
+using static UnityEngine.InputSystem.InputAction;
 
 public class EscMenuController : MonoBehaviour
 {
+    [SerializeField] private InputAction pauseAction;
+
     public GameObject pauseMenuPanel;
     public GameObject settingsMenuPanel;
```

Comment:
these should probably also be private SerializeFields (with properties to access them in other classes)